### PR TITLE
removed .vs code

### PR DESCRIPTION
**Pull request by renuka7**

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -280,17 +280,17 @@ Provides the template integration for [JSX Templates][JSX_WEBSITE].
...
-Provides the build system integration for [Ember CLI][EMBER_CLI_WEBSITE].
+Provides the build system integration for [Ember CLI][https://ember-cli.com/].
 
 ## Public Websites
 
-### [@css-blocks/website][WEBSITE]
+### [@css-blocks/website][http://css-blocks.com]
 The public-facing website and hosted Typedoc API documentation hosted at [www.css-blocks.com][CSS_BLOCKS_WEBSITE]. Built as a React app and dog-foods our JSX/Webpack integrations.
 
 ### [@css-blocks/playground][PLAYGROUND]
```
Comment (kaleiukea) [inline ARCHITECTURE.md]: [quorum-consortium-network-in-azure-marketplace.pdf](https://github.com/linkedin/css-blocks/files/1959244/quorum-consortium-network-in-azure-marketplace.pdf)

**Comment by ForsakenHarmony** [comment]
wtf is this pull request?

**Comment by renuka7** [comment]
Just tried

**Comment by chriseppstein** [comment]
@renuka7 This pull request doesn't clearly explain what it's trying to accomplish and the changes don't appear to be necessary. If you have a specific problem, please start by filing a new issue explaining the problem.

@ForsakenHarmony I agree this is PR confusing, but I'd ask you to find a kinder way to say that in the future.

**Pull request closed without merging**